gigadb/app/tools/readme-generator/README.md

```diff
@@ -79,11 +79,14 @@ or `port install bats-core`). Then run:
 # Ensure you are in gigadb/app/tools/readme-generator directory
 $ bats tests
  ✓ create readme file
- ✓ create readme file and copy to wasabi
+ ✓ create one readme file using batch mode
+ ✓ create two readme files using batch mode
+
+3 tests, 0 failures
```

Comment:
When I run the bats tests, I got the this output:
```
% bats tests
createReadme.bats
 ✓ create readme file
 ✓ check does not create readme with invalid doi and exits
 ✓ create one readme file using batch mode
 ✓ create two readme files using batch mode

4 tests, 0 failures
```

Reply:
Updated output of bats tests now.